src/transformers/tokenization_utils_base.py

```diff
@@ -2222,6 +2222,11 @@ def _get_padding_truncation_strategies(
                 padding_strategy = PaddingStrategy.MAX_LENGTH
         elif padding is not False:
             if padding is True:
+                if verbose:
+                    if max_length is not None:
+                        warnings.warn("`max_length` is ignored when `padding`=`True`.")
+                    if old_pad_to_max_length is not False:
+                        warnings.warn("`pad_to_max_length` is ignored when `padding`=`True`.")
```

Comment:
This argument is `True` when the following kwarg was passed as `True`: `pad_to_max_length`. Could you edit the warning to show that?

Reply:
I updated the line.
Is it what you intended?

Reply:
Thank you!